<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimalInfos</class>
 <widget class="QWidget" name="AnimalInfos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>901</width>
     <height>631</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="ADDNEW">
    <attribute name="title">
     <string>Add New Animal</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_addAnimal">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>530</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="species">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Color_6">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>352</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Color</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>black</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>brown</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>grey</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>white</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mixed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spotted</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>stripped</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_53">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_56">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>240</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>age</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_52">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>species</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_61">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>480</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>picture location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>282</x>
       <y>155</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_59">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>405</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>coat length</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Size_6">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>320</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Size</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>small</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>large</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_58">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>320</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>size</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combobox_Age_6">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>240</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Age</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>puppy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>adult</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_Gender_6">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>280</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Gender</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>female</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>male</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_60">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>360</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>color</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_activeLevel_6">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>440</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Active level</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>low</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>high</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_107">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>shelter ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="shelterID">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_54">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>440</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>energy level</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_55">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>280</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>gender</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_CoatLength_6">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>400</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Coat length</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>no hair</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>short hair</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mid-length hair</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>long hair</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="otherpicaddress">
     <property name="geometry">
      <rect>
       <x>288</x>
       <y>480</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_status">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>530</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>breeds</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_breeds">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>200</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="DELETE">
    <attribute name="title">
     <string>Delete Existing animal</string>
    </attribute>
    <widget class="QLineEdit" name="delete_input_id">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>400</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>400</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Please enter the animal ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteanimal">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>440</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="animalTable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>801</width>
       <height>341</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>8</number>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ShelterID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Species</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Breed</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Gender</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Age</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton_pre">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>510</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_next">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>510</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>26</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QToolButton" name="toolButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Adobe Gothic Std</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>PetFinder</string>
          </property>
          <property name="icon">
           <iconset resource="../project/Resources.qrc">
            <normaloff>:/imgs/CommunityIcons_home.png</normaloff>:/imgs/CommunityIcons_home.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>41</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Adobe Ming Std</family>
          <pointsize>20</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Animal Info</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_11">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9"/>
    </item>
    <item>
     <spacer name="horizontalSpacer_12">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../project/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
